cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -993,7 +993,7 @@ class BudgetNotificationWithSubscribersSchema(BaseSchema):
         required=True,
         metadata={"update_policy": UpdatePolicy.SUPPORTED},
     )
-
+    trigger_fleet_stop = fields.Bool(metadata={"update_policy": UpdatePolicy.SUPPORTED})
```

Comment:
Do we want to add a validation to only support it in cluster budget?

Reply:
That makes sense. Schema validation added!